bb8/src/api.rs

```diff
@@ -260,6 +271,15 @@ impl<M: ManageConnection> Builder<M> {
         self
     }
 
+    /// Sets the queue strategy to be used by the pool
+    ///
+    /// Defaults to `LIFO`.
```

Comment:
```suggestion
    /// Defaults to `FIFO`.
```
according to the code above